lib/baza-rb.rb

```diff
@@ -706,30 +706,29 @@ def checked(ret, allowed = [200])
       @loog.debug(log)
       return ret
     end
-    @loog.debug("#{log}\n  #{(ret.headers || {}).map { |k, v| "#{k}: #{v}" }.join("\n  ")}")
     headers = ret.headers || {}
-    msg = [
-      "Invalid response code ##{ret.code} ",
-      "at #{mtd} #{url}",
-      headers['X-Zerocracy-Flash'] ? " (#{headers['X-Zerocracy-Flash'].inspect})" : ''
-    ].join
+    if headers.any?
+      headers_log = headers.map { |k, v| "  #{k}: #{v}" }.join("\n")
```

Comment:
@paveltrty read this: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html